srv/modules/runners/changed.py

```diff
@@ -183,56 +241,71 @@ def rgw():
     """
     Returns whether RadosGW configuration changed
     """
-    return requires_conf_change('rgw')
+    srv_rgw = Role(role_name='rgw')
+    return requires_conf_change(role=srv_rgw)
```

Comment:
This is possibly silly nitpicking on my part, but why not shrink this (and subsequent similar lines) to `return requires_conf_change(role=Role(role_name='rgw'))`?

Reply:
good point, pushing..